Exercism/javascript/two-fer/two-fer.js

```diff
@@ -0,0 +1,33 @@
+import { always, curry, ifElse, isNil } from 'ramda';
+
+/* Version 1 */
+
+// twoFer :: String -> String
+// const twoFer = name => {
+//   if (!name) {
+//     return 'One for you, one for me.';
+//   }
+
+//   return `One for ${name}, one for me.`;
+// };
+
+/* Version 2 */
+
+// twoFer :: String -> String
+// const twoFer = name =>
+  // name ? `One for ${name}, one for me.` : 'One for you, one for me.';
+
+/* Version 3 */
+
+// interpolate :: String -> String -> String -> String
+const interpolate = curry((before, after, value) => (before + value + after));
```

Comment:
This is definitely naive, but I think something like this would be a more functional approach to string formatting.